applications/ShapeOptimizationApplication/test_examples/.gitignore

```diff
@@ -1 +1 @@
-**/*.csv
\ No newline at end of file
+**/Optimization_Results/
```

Comment:
Why is this necessary?

Reply:
The result name of the vtk files is not anymore "design_history.vtk" but "MODELPARTNAME.vtk". Tto avoid accidental commits of the result files, i think its the easiest solution to just exclude the complete optimization results directory from git.

Reply:
Ah ok. But note that the name of the results directory is an option in the optimization settings under "output_directory". That would make this gitignore obsolete. But I think it even makes sense to remove that option and fix the name.

Another thing is the filename of the design results. Note, that this is also an option. However, this PR changes that logic for the VTK-output so that the name of the corresponding sub-mdpa is used. But also here, I think, we can remove that option and use the name of the sub-mdpa in all cases.

Last not least we might fix the "response_log_filename" so that this also drops out from the settings. By that, the output options would really reduce to the actually necessary things like nodal_results.

@armingeiser What do you think?

Reply:
:+1: completely agree. I think we never used these options since we have them.